dev/docs/README.md

```diff
@@ -8,4 +8,4 @@ source.
 * **Main site: [flutter.io](https://flutter.io/)**
 * [Install](https://flutter.io/setup/)
 * [Get started](https://flutter.io/getting-started/)
-* [Contribute](CONTRIBUTING.md)
+* [Contribute](https://github.com/flutter/flutter/blob/master/CONTRIBUTING.md#contributing-to-flutter)
```

Comment:
nit: remove the `#contributing-to-flutter` - since that's the headline for the doc it seems to provide no value to link to the header directly?

Reply:
It actually does have an effect: it causes the page to scroll down to the heading, so it looks like a doc rather than a GitHub page.

Reply:
Sounds good.